Jenkinsfile

```diff
@@ -54,15 +54,20 @@ pipeline {
                     steps {
                         echo "Running nosetests on Python 2.7"
                         sh 'python2 -m pip install -U .'
-                        sh "python2 setup.py nosetests --with-xunitmp --xunitmp-file=nosetests_py27.xml"
+                        sh 'python2 -m coverage run --source="${KATPACKAGE}" -m nose --with-xunitmp --xunitmp-file=nosetests_py27.xml'
+                        sh 'python2 -m coverage xml -o coverage_27.xml'
+                        sh 'python2 -m coverage report -m --skip-covered'
+
                     }
                 }
 
                 stage ('py36') {
                     steps {
-                         echo "Running nosetests on Python 3.6"
-                         sh 'python3 -m pip install -U .'
-                         sh "python3 setup.py nosetests --with-xunitmp --xunitmp-file=nosetests_py36.xml"
+                        echo "Running nosetests on Python 3.6"
+                        sh 'python3 -m pip install -U .'
+                        sh 'python3 -m coverage run --source="${KATPACKAGE}" -m nose --with-xunitmp --xunitmp-file=nosetests_py36.xml'
```

Comment:
Confirmed `katcp` is baked into the image.

![image](https://user-images.githubusercontent.com/7910856/71350140-4eb55a80-2579-11ea-9de7-f951003e510e.png)

Reply:
thanks. It was not there when I was doing testing last week and hence the addition. It has been removed.